dpdata/dftb_plus/inout.py

```diff
@@ -0,0 +1,45 @@
+from typing import Tuple
+
+import numpy as np
+
+
+def read_dftb_plus(fn: str) -> Tuple[str, np.ndarray, float, np.ndarray]:
```

Comment:
DFTB+ has input and output files separately, so I suggest not merging them into one file. Instead, pass a directory or two file arguments.